### PR TITLE
doc: Add 2.1.2 to main

### PR DESCRIPTION
**Pull request by umapraseeda**
Add 2.1.2 docs to main

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

- [x] Add release notes (with highlights)

### CODE REVIEW

**Review by peknis — APPROVED** [review]

File: doc/nrf/releases/release-notes-2.1.2.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This minor release adds the following changes on top of the :ref:`nRF Connect SDK v2.1.0 <ncs_release_notes_210>`:
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-2.1.2.rst:19]: See edits to #9114

File: doc/nrf/releases/release-notes-2.1.2.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+Bluetooth mesh
+--------------
+
+* Fixed bugs that appeared when using extended advertiser.
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``71ef669ea4a73495b255f27024bcd5d542bf038c``.
+This is the same commit ID as the one used for |NCS| :ref:`v2.1.0 <ncs_release_notes_210>`.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-2.1.2.rst:93]: See edits to #9114

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-2.1.2.rst (hunk 1 above)
Comment (umapraseeda) [reply doc/nrf/releases/release-notes-2.1.2.rst]: applied here too.

File: doc/nrf/releases/release-notes-2.1.2.rst (hunk 2 above)
Comment (umapraseeda) [reply doc/nrf/releases/release-notes-2.1.2.rst:93]: Applied edits from 9114.

**Review by greg-fer — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Comment by rlubos** [comment]
Aborted the full CI since it's doc only (apparently the CI did not detect the label on rerun)

**Pull request merged**